pkg/crc/logging/logging.go

```diff
@@ -1,14 +1,13 @@
 package logging
 
 import (
-	"github.com/pkg/errors"
-	"github.com/sirupsen/logrus"
-	"io"
 	"io/ioutil"
 	"os"
 	"path/filepath"
 
 	"github.com/code-ready/crc/pkg/crc/constants"
+	"github.com/pkg/errors"
```

Comment:
pkg/errors?

Reply:
This is only used in one place, can probably be changed to use crc `errors` package
https://github.com/code-ready/crc/blob/6a3e401528bb982ddf1f3a5d26995b3d792f706b/pkg/crc/logging/logging.go#L82